src/Testcontainers.FirebirdSql/FirebirdSqlBuilder.cs

```diff
@@ -0,0 +1,170 @@
+namespace Testcontainers.FirebirdSql;
+
+/// <inheritdoc cref="ContainerBuilder{TBuilderEntity, TContainerEntity, TConfigurationEntity}" />
+[PublicAPI]
+public class FirebirdSqlBuilder : ContainerBuilder<FirebirdSqlBuilder, FirebirdSqlContainer, FirebirdSqlConfiguration>
+{
+    public const string FirebirdSqlImage = "jacobalberty/firebird";
+
+    public const ushort FirebirdSqlPort = 3050;
+
+    public const string DefaultDatabase = "test";
+    public const string DefaultUsername = "test";
+    public const string DefaultPassword = "test";
+    public const string FirebirdSysdba = "sysdba";
+    public const string DefaultSysdbaPassword = "masterkey";
+
+    /// <summary>
+    /// Initializes a new instance of the <see cref="FirebirdSqlBuilder" /> class.
+    /// </summary>
+    public FirebirdSqlBuilder()
+        : this(new FirebirdSqlConfiguration())
+    {
+        DockerResourceConfiguration = Init().DockerResourceConfiguration;
+    }
+
+    /// <summary>
+    /// Initializes a new instance of the <see cref="FirebirdSqlBuilder" /> class.
+    /// </summary>
+    /// <param name="resourceConfiguration">The Docker resource configuration.</param>
+    public FirebirdSqlBuilder(FirebirdSqlConfiguration dockerResourceConfiguration)
+        : base(dockerResourceConfiguration)
+    {
+        DockerResourceConfiguration = dockerResourceConfiguration;
+    }
+
+    /// <inheritdoc />
+    protected override FirebirdSqlConfiguration DockerResourceConfiguration { get; }
+
+
+    /// <summary>
+    /// Sets the FirebirdSql username.
+    /// </summary>
+    /// <remarks>
+    /// Can be set to 'sysdba'. In that case please use <see cref="WithSysdbaPassword" /> to set the password to connect with.
+    /// </remarks>
+    /// <param name="username">The FirebirdSql username.</param>
+    /// <returns>A configured instance of <see cref="FirebirdSqlBuilder" />.</returns>
+    public FirebirdSqlBuilder WithUsername(string username)
+    {
+        var builder = Merge(DockerResourceConfiguration, new(username: username));
+        if (username.ToLower() != FirebirdSysdba)
+        {
+            builder = builder
+                .WithEnvironment("FIREBIRD_USER", username);
+        }
+
+        return builder;
+    }
```

Comment:
If a developer uses `sysdba`, the method will ignore it. Shouldn't we check this in the `Validation()` method? Or, we could use a similar approach as we are using in the [MySqlBuilder](https://github.com/testcontainers/testcontainers-dotnet/blob/e3be24f8256c21be4525e8abae623695c8bb7fb2/src/Testcontainers.MySql/MySqlBuilder.cs#L50-L71). I think I favor this. It makes the configuration simpler. WDYT?

Reply:
I see your point, I think the way it is handled in the MySqlBuilder works great, I will apply that.